include/gnc_constants.hpp

```diff
@@ -0,0 +1,52 @@
+//
+// include/gnc_constants.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+/* - NOTICE - Values in this file should be seldomly edited and have hidden
+ *            interdependancies. */
+
+#ifndef PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+#define PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+
+#include "lin.hpp"
+
+namespace gnc {
+namespace constant {
+
+/** Initial GPS week number that serves as the 'PAN epoch'. Some of the
+ *  constants below depend on this number. */
+constexpr static unsigned int init_gps_week_number = 2045;
+
+/** Earth's intertial rotation rate about the z axis in the ECEF frame.
+ *  NOTE: Pulled from PSim. */
+constexpr static double earth_rate_ecef_z = 1.0e-04 * 0.729211585530000;
+
+/** Approximate precssion rate of Earth's Axis in radians per second.
+ *  NOTE: Pulled from PSim and needs to be updated with a new GPS week. */
```

Comment:
Can you create an issue tracking these "required updates" so that we don't lose them

Reply:
Yep I was going to flush all of this out and potentially invest the time in writing a script to autocoder the required constants and unit tests. Wanted to discuss this stuff in more detail, however, and we'll probably get to that Thursday would be my guess.